server/static/js/course.js

```diff
@@ -179,8 +179,16 @@ function(RmcBackbone, $, _, _s, ratings, __, util, jqSlide, _prof, toastr,
         }
       };
 
-      $.ajax('/api/v1/user/shortlist/' + this.courseModel.id, { type: 'PUT' })
+      var _user = require('user');
+      if(_user.getCurrentUser() === null) {
+        // Render the login modal
+        $('.email-login-modal').modal('show');
```

Comment:
Could you make this a function inside of server/static/js/sign_in.js so it can be reused? Also, when you move it into sign_in.js you can namespace the selector by doing `emailLoginModalView.$('.email-login-modal').modal('show')`